### DISCUSSION TITLE
blog/2023-06-19-causal-inference-with-change-scores/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#20] [Comments]
# blog/2023-06-19-causal-inference-with-change-scores/

Part 8 of the GLM and causal inference series.

https://solomonkurz.netlify.app/blog/2023-06-19-causal-inference-with-change-scores/

**Comment by datarichard** [comment]
Nice exposition and summary of the literature around this thorny issue. One thing I didn't follow is that Rubin says the change score approach is an unbiased estimate of the ATE, but without the pre covariate it is not protected against RTM. So wouldn't the presence of RTM disqualify the change score as an unbiased estimate? Is Rubin mistaken?

**Reply by ASKurz** [reply]
@datarichard, my understanding is RTM in this context has more to do with efficiency than bias.

**Reply by datarichard** [reply]
ok yes, I see how that might make sense in the context of an RCT since any pre-existing differences will be random and so any RTM will also be random.

**Comment by datarichard** [comment]
Another dumb question: My understanding is that one of the effects of using change/gain scores is to remove the pre-existing variation between subjects. Adding the pre scores as a covariate will also do this (is that correct??). So the ANCOVA-change model seems to be removing the pre-existing subject variation twice - is this a problem here? It doesn't seem to be since both ANCOVA models give the same result. Is this always the case? (I'm beginning to realise all the causal inference I learnt in Psych101 is a big can of worms!!)

**Reply by ASKurz** [reply]
@datarichard, I don't think I know what you mean by "pre-existing subject variation." Can you say more or provide a reference?

**Reply by datarichard** [reply]
If I understand correctly, the gain scores express the change from baseline, so any between-subject baseline differences are removed. I can't give you an academic reference for my concern - it was raised in the final sentence of this blog which is discussing a RM ANOVA: https://www.theanalysisfactor.com/pre-post-data-repeated-measures/

But I think my concern is misplaced, since you are not discussing a repeated measures analysis here, but rather change scores, which as was just mentioned do not represent the baseline variation anyway. So the ANCOVA-change model doesn't include the baseline variation twice, and the maths shows it is equivalent to the ANCOVA-post model anyway...

Anyway, thanks for the blog post because I'm learning a lot (that I thought I already knew!)

**Reply by ASKurz** [reply]
I looked at the opening of the Analysis Factor post you linked to, and I believe you're correct. Their focus on the RN ANOVA is different from what I'm focusing on in this post. However if I understand the RN ANOVA correctly (which I may well not because I have not used one in about 10 years), it corresponds to the DiD analysis I briefly covered at the end of this post. The reason we wouldn't do a DiD-style analysis with RCT data is because it's not a great idea to model group differences at baseline, and I believe that's what would happen with an RN ANOVA.

**Comment by ASKurz** [comment]
For a discussion of how to think about standardized mean difference effect sizes for change-scores, see Morris (2008; https://doi.org/10.1177/1094428106291059).
